src/select.rs

```diff
@@ -77,24 +85,52 @@ where
         }
         <Self as PromptInteraction<T>>::interact(self)
     }
+
+    /// Enable the filter mode
+    pub fn enable_filter(mut self) -> Self {
```

Comment:
🤔 Hm, all those builder methods are not verbs except for the final `interact`, so it's better to have something like `filter_mode` or `search_mode` instead:

```rust
select().item("a").item("b").search_mode().interact();
```

Reply:
Yes it makes more sense for sure 👍 I switched to filter_mode() to match with the `input_filter`